File: hira/hira/hirastage_stack.py
import aws_cdk

from aws_cdk import (
    Stack,
    pipelines,
    Stage,

)
import aws_cdk as cdk
from .hira_stack import HiraStack

from constructs import Construct


class HirastageStack(Stage):

    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
        super().__init__(scope, construct_id, **kwargs)

        # Instantiate web crawler stack
        self.stage = HiraStack(self, "hiraaziz3stack")

File: hira4sprint/lib/hirastagestack.ts
import { Stack, StackProps, Stage} from 'aws-cdk-lib';
import * as lambda from 'aws-cdk-lib/aws-lambda';
import { Construct } from 'constructs';
import {Hira4SprintStack} from './hira4sprint-stack'
import * as cdk from 'aws-cdk-lib';


export class Hirastagestack extends Stage {
    constructor(scope: Construct, id: string, props?: StackProps) {
      super(scope, id, props);

        // Instantiate Stack      
         const hira4stack = new Hira4SprintStack(this,"HiraStack")
    }
}